app.rb

```diff
@@ -14,26 +14,31 @@ class AbodenB < Sinatra::Base
   get "/signup" do
     erb :signup
   end
+
   post "/signup" do
     User.create(username: params[:username], email: params[:email], password: params[:password])
     redirect "/profile"
   end
+
   get "/profile" do
     erb :profile
-
+  end
 
   get "/listings/new" do
     erb :listings_new
   end
 
   post "/listings/new" do
```

Comment:
within post request, will these variables be used again? If not it might be ok to just pass the params directly do the self.create method to keep the controller skinny

Reply:
Good shout, updated!